### PR TITLE
Remove the -test switch from ramfs.go and fix README.md

### PR DESCRIPTION
**Pull request by rminnich**
We've come too far down the road of working initramfs for the
-test switch to be meaningful any more. A pity, but the kinds of
things it has to do to make the test useful are just more than
linux can stand.

remove the -test switch, and a bunch of code; fix README.md.

While we're at it, clean up guessgopath to just die if GOPATH is
not set.

I'm cleaning this up because it was causing trouble for new users.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: scripts/ramfs.go [hunk 1]
```diff
@@ -559,32 +558,7 @@ func guessgopath() {
...
-		}
-		config.Gopath = c
-		log.Printf("Guessing %v as GOPATH", c)
-		os.Setenv("GOPATH", c)
-		return
-	}
-	config.Fail = true
-	log.Printf("GOPATH was not set, and I can't see a u-root-like name in %v", cwd)
-	return
+	log.Fatalf("You have to set GOPATH, which is typically ~/go")
```
Comment (rjoleary) [inline scripts/ramfs.go]: As of Go 1.8, "$HOME/go" is automatically used if GOPATH is not set.

Comment (rminnich) [reply scripts/ramfs.go]: Thanks Ryan, this allows me to fix BOTH ramfs.go and bb.go.

one thing I've realized in this project how much go has changed in 6 years!

Comment (rminnich) [reply scripts/ramfs.go]: Ah, right, I use GOPATH for a few other things, I'll leave it in for now.

Comment (rminnich) [reply scripts/ramfs.go]: So there will be a future fix to accomodate this change.

**Comment by ryukinix** [comment]
Just to saying @rminnich I saw only now that you forgot to remove the `RUN` script or edit it to fit with this removal. Since we not have more the `-test` flag, I think we should change that too.

**Pull request merged**